rcsb/exdb/seq/ReferenceSequenceAnnotationProvider.py

```diff
@@ -170,6 +169,9 @@ def __fetchSiftsSummaryProvider(self, cfgOb, configName, **kwargs):
         logger.debug("ssP %r %r", srcDirPath, cacheDirPath)
         ssP = SiftsSummaryProvider(srcDirPath=srcDirPath, cacheDirPath=cacheDirPath, useCache=useCache, abbreviated=abbreviated, cacheKwargs=cacheKwargs)
         ok = ssP.testCache()
+        if not ok:
+            logger.error("Failed to refetch SIFTS summary data using srcDirPath %s, cacheDirPath %s", srcDirPath, cacheDirPath)
+            return None
```

Comment:
Will this force the process to exit with a non-0 status?

Reply:
If I simply try to Raise a ValueError here (instead of `return None`), that itself doesn't cause the luigi workflow to exit. It was necessary to pass the `ok` status through the established channels in order to trigger the exit. The channels are all generally sound, but this one was an exception that wasn't being caught properly. But now we can rely on it.

Reply:
That sounds great, thanks for explaining!